packages/fluentui/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -188,7 +188,6 @@ const ToolbarItem: React.FC<WithAsProp<ToolbarItemProps>> &
     if (menu) {
       if (doesNodeContainClick(menuRef.current, e.nativeEvent as MouseEvent, context.target)) {
         trySetMenuOpen(false, e);
-        _.invoke(itemRef.current, 'focus');
```

Comment:
Should we remove the same thing in `closeMenuAndFocusTrigger`?

Reply:
no, if that is the one used for handling ESC key, we need to focus trigger in that case

Reply:
to @jurokapsiar: yes "closeMenuAndFocusTrigger" is calling when ESC key is executed :)